fizzbuzz.py

```diff
@@ -0,0 +1,9 @@
+def fizzbuzz(input):
+    if input % 3 == 0 and input % 5 == 0:
+        return 'fizzbuzz'
+    else if input % 3 == 0:
+        return 'fizz'
+    else if input % 5 == 0:
+        return buzz
```

Comment:
```suggestion
        return 'buzz'
```

Reply:
correct

Reply:
```suggestion
        return buzz
``` Do you want to enclose the String with quotes?

Reply:
correct

Reply:
Here really need quotes